tests/functionnal/vagrant/common.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -xe
+
+apt-get -y update
+# install vagrant
+wget https://releases.hashicorp.com/vagrant/1.9.6/vagrant_1.9.6_x86_64.deb -O vagrant.deb
+dpkg -i vagrant.deb
+apt-get -f -y  install
+# install vbox
+wget http://download.virtualbox.org/virtualbox/5.1.22/virtualbox-5.1_5.1.22-115126~Debian~jessie_amd64.deb -O vbox.deb
+dpkg -i vbox.deb || true
+apt-get -f -y  install
+
+# install enos dependencies
+apt-get install -y libffi-dev libssl-dev python-dev python-pip python-virtualenv
```

Comment:
These dependencies have already been described in `tests/functionnal/packer/packer_templates/scripts/enos/install_deps.sh`. I don't know if it's redundant or not.